tests/test_slurm.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from IPython.testing.globalipapp import start_ipython
+
+# pylint: disable=redefined-outer-name
+
+
+@pytest.fixture(scope="session")
+def session_ip():
+    """Get an IPython shell"""
+    yield start_ipython()
+
+
+@pytest.fixture(scope="function")
+def ip(session_ip):
+    """Prepare IPython shell for running unit tests"""
+    session_ip.run_cell(raw_cell="import qililab.slurm")
+    yield session_ip
+
+
+def test_submit_job(ip):
+    ip.run_cell(raw_cell="a=1\nb=1")
+    ip.run_cell_magic(
+        magic_name="submit_job",
+        line="-o results -d debug -l slurm_job_data_tests -n unit_test -e local",
+        cell="results = a+b ",
+    )
+    assert ip.user_global_ns["results"].result() == 2
```

Comment:
Can you also add asserts checking that the arguments work? For example: check that files are created inside the `slurm_job_data_tests` folder.